### DISCUSSION TITLE
DT/HT mod icons changing color based on speed increase/decrease

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15262] [Ideas]
I could definitely understand why people might not like this idea as the colors of the default DT/HT mod icons wouldn't be consistent with the other difficulty increasing/decreasing mods, but I still wanted to put this here since I thought it would be an interesting idea. Also, this is a very rough idea of what I personally think it should look like (I'm not a graphics designer) and it is most certainly subject to change based on what people think.

![HT   DT coloured icon proposal](https://user-images.githubusercontent.com/74088315/138582571-ea0010e8-56d9-4841-8118-2b15b737e88b.png)

The reason why I even thought of this idea in the first place is because HT can range anywhere from 0.5-0.99x speed and DT can range anywhere from 1.01-2x speed; in my opinion, it just feels weird for the mod icons to remain the same when there can be such drastic changes in the actual effects of the mods and it would be nice to have some sort of visual indicator to distinguish between varying speeds without having to hover over the mod icons.

**Comment by peppy** [comment]
I'd prefer the rate is shown in the corner of the mod icon or something. I don't think we want to use colours to signify this.
